### PR TITLE
Increases file descriptor limit for root to 100K in start_tpu.sh.

### PR DESCRIPTION
**Pull request by ruomingp**

### CODE REVIEW

File: axlearn/cloud/gcp/scripts/start_tpu.sh [hunk 1]
```diff
@@ -12,6 +12,9 @@ echo "=== AXLearn start_tpu.sh ==="
 # Random sleep to prevent all TPU-VMs overwhelming pypi etc for large slices.
 sleep $((1 + $RANDOM % 30))
 
+sudo sh -c "echo 'root soft nofile 100000' >> /etc/security/limits.conf"
```
Comment (tgunter) [inline axlearn/cloud/gcp/scripts/start_tpu.sh:15]: nit: Is it worth a comment to explain?

**Review by tgunter — APPROVED** [review]

**Review by samos123 — APPROVED** [review]

File: axlearn/cloud/gcp/scripts/start_tpu.sh (hunk 1 above)
Comment (ruomingp) [reply axlearn/cloud/gcp/scripts/start_tpu.sh:16]: Done.

File: axlearn/cloud/gcp/scripts/start_tpu.sh [hunk 2]
```diff
@@ -12,6 +12,10 @@ echo "=== AXLearn start_tpu.sh ==="
 # Random sleep to prevent all TPU-VMs overwhelming pypi etc for large slices.
 sleep $((1 + $RANDOM % 30))
 
+# Increase file descriptor limits for `root` to avoid "Too many open files" errors.
```
Comment (samos123) [inline axlearn/cloud/gcp/scripts/start_tpu.sh:15]: nit: the change the comment to an echo statement so someone reading the logs knows what's going on. That would have helped in catching whether the correct startup script was run as well.

Comment (ruomingp) [reply axlearn/cloud/gcp/scripts/start_tpu.sh]: Good point! Done.

**Pull request closed without merging**